src/resources/views/crud/create.blade.php

```diff
@@ -15,10 +15,10 @@
 	<section class="container-fluid">
 	  <h2>
         <span class="text-capitalize">{!! $crud->getHeading() ?? $crud->entity_name_plural !!}</span>
-        <small>{!! $crud->getSubheading() ?? trans('backpack::crud.add').' '.$crud->entity_name !!}.</small>
+        <small class="fs-6">{!! $crud->getSubheading() ?? trans('backpack::crud.add').' '.$crud->entity_name !!}.</small>
 
         @if ($crud->hasAccess('list'))
-          <small><a href="{{ url($crud->route) }}" class="d-print-none font-sm"><i class="la la-angle-double-{{ config('backpack.base.html_direction') == 'rtl' ? 'right' : 'left' }}"></i> {{ trans('backpack::crud.back_to_all') }} <span>{{ $crud->entity_name_plural }}</span></a></small>
+          <small class="fs-6"><a href="{{ url($crud->route) }}" class="d-print-none font-sm"><i class="la la-angle-double-{{ config('backpack.base.html_direction') == 'rtl' ? 'right' : 'left' }}"></i> {{ trans('backpack::crud.back_to_all') }} <span>{{ $crud->entity_name_plural }}</span></a></small>
```

Comment:
I see these heading classes are added to ALL operation views. That tells me... we could probably do them as a CSS rule, instead of adding helper classes. Moving this to the CSS file would help us:
- undo the breaking changes in 4-5 blade files (`create`, `update`, `list`, `show`, `reorder`)
- make the headings look good by default, in any custom operations devs might have created
- make the headings look good by default in other packages of ours

I think a good test here would be to look at LogManager:
![CleanShot 2023-02-26 at 10 44 21](https://user-images.githubusercontent.com/1032474/221400812-c88c1302-e846-4be5-ae9a-309ca115d576.png)

If: 
- all CRUD operations look good, without changes to blade files... 
- LogManager looks good, without any changes to the package itself... 

it means we've upgraded this bit to Bootstrap v5 as a non-breaking change.

Reply:
I'd add a small margin-left before the subheading... it's too close right now I think:

<img width="1498" alt="CleanShot 2023-02-27 at 14 42 20@2x" src="https://user-images.githubusercontent.com/1032474/221566713-6e2f4488-77bc-43da-a359-245e3ac7694c.png">

But the operations have a small padding before the subheading:
<img width="1495" alt="CleanShot 2023-02-27 at 14 42 43@2x" src="https://user-images.githubusercontent.com/1032474/221566844-f61c67cf-bf71-42a2-a8a3-87df8ae81454.png">

Which makes me wonder... are they using the same CSS?

Reply:
Hmmm yes they are... so why do they look slightly different then?